TIL/topics/java_basic/2025.06.17 .md

```
@@ -0,0 +1,78 @@
+
+# TIL(Today I Learned)
+
+## Test Code
+ - assertEquals(); / assertNotEquals();
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)